README.md

```diff
@@ -75,3 +75,21 @@ The tests are configured through `.test_config.yaml` file. Copy `.test_config.ya
 
   - `turso_db_path`: URL for turso database used during the tests. It should be a dedicated database once it's cleared after each test. Follow steps from [Running the Project/Turso](#Turso) to generate a Turso db path. For this database, it's useful to generate a non-expiration token with `turso db tokens create --expiration none <turso_test_db_name>`
   - `skip_turso_tests`: Used to skip Turso related tests
+
+### Running tests against a locally-running sqld instance
+
+An easy way to get an sqld server running locally is to clone
+[hrana-test-server](https://github.com/libsql/hrana-test-server)
+and run the
+[test_server](https://github.com/libsql/hrana-test-server/blob/main/test_server.py)
+python script.
+
+Then, in `libsql-shell-go`, edit your `test_config.yaml` file in by adding the url
+of the locally running sqld server under `turso_db_path`. Once you have this configuration
+and the sqld instance running simply run the tests normally:
```

Comment:
missing comment about setting `skip_sqld_tests` to `false`

Reply:
done